test/unit/test_fracture_importer.py

```diff
@@ -251,5 +251,95 @@ def test_create_fracture(self):
         self.assertTrue(f.p[2].min() == 0)
 
 
+class TestImportDFN(unittest.TestCase):
+    def test_one_fracture(self):
+        p = np.array([0, 0, 1, 1])
+        f = np.hstack((0, p))
+        file_name = "frac.csv"
+        np.savetxt(file_name, f, delimiter=",")
+
+        network = pp.fracture_importer.network_2d_from_csv(file_name, skip_header=0)
+
+        mesh_args = {"mesh_size_frac": 0.3, "mesh_size_bound": 0.3}
+        gb = network.mesh(mesh_args, dfn=True)
+
+        self.assertTrue(gb.num_cells() == 5)
```

Comment:
Do these tests assume that the mesh produced by gmsh does not change with changing gmsh versions? I am no fan of that, but at the very least, you should document it.

Reply:
I was not sure what to test, they might change for different version of gmsh but we are speaking of 1d grids made of straight lines. I guess this part in gmsh is quite stable. I add a comment

Reply:
I changed the test to be less dependent on gmsh. @keileg let me know what you think